shuup/core/order_creator/_creator.py

```diff
@@ -317,7 +314,4 @@ def create_order(self, order_source):
         order.save()
         order = self.finalize_creation(order, order_source)
         order_creator_finished.send(sender=type(self), order=order, source=order_source)
-        # reset product prices
-        for line in order.lines.exclude(product_id=None):
-            context_cache.bump_cache_for_product(line.product, shop=order.shop)
```

Comment:
Feel free also other option here would be to use "values_list" which again should be faster. I guess the point here is that we need to bump the cache for ordered products since for example the orderability might have changed etc..

Can also be in the signal handlerfor sure.

Reply:
Moved this to `order_creator_finished`